src/generated/resources/data/occultism/recipe/ritual/possess_random_animal_special.json

```diff
@@ -0,0 +1,53 @@
+{
+  "type": "occultism:ritual",
+  "activation_item": {
+    "item": "occultism:book_of_binding_bound_afrit"
+  },
+  "duration": 60,
+  "entity_tag_to_summon": "occultism:random_animals_special",
+  "ingredients": [
+    {
+      "item": "minecraft:white_wool"
```

Comment:
These recipes that require a bunch of different yet similar items are no fun as a player and really don't add anything to the experience (Budding Amethyst recipes also suffers from this, making the player sit and stare at a budding amethyst for an hour to get all the different growth stages.). Using the wool tag here would make more sense.

Reply:
I will change this recipe a little, thanks for the feedback

Reply:
changed the specific wools to 2 any wool and the mushrooms blocks to 2 any mushrooms (not full block)